examples/grids/grid.jl

```diff
@@ -25,7 +25,7 @@ using Raven
 using WriteVTK
 using StaticArrays
 
-# here the level is a tuple to 
```

Comment:
Typically I would not include this minor formatting change in this commit.  It is unrealted to the purpose of the commit.